lib/raw_net_capture.rb

```diff
@@ -24,7 +24,7 @@ def reset
 end
 
 class RawHTTPCapture < StringIO
-  attr_reader :raw_received, :raw_sent
+  attr_reader :raw_received, :raw_sent, :transactions
```

Comment:
Does it make sense to keep `raw_received` and `raw_sent` at top level? Maybe just have `transactions` and start with an empty transaction (or zero transactions and create one as soon as data is first sent)